ansible_base/jwt_consumer/common/auth.py

```diff
@@ -56,6 +70,8 @@ def parse_jwt_token(self, request):
                 "is_superuser": validated_body["is_superuser"],
             },
         )
+        delta_ms = (time.time() - start) * 1000
+        logger.debug(f'Getting/saving user took {delta_ms:.0f} (ms)')
```

Comment:
It would be better to refactor this code into a method and them apply the decorator the same as elsewhere.